src/Microsoft.TestPlatform.CrossPlatEngine/Utilities/TestCaseFilterDeterminer.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.Utilities
+{
+    internal class TestCaseFilterDeterminer
+    {
+        internal static string ShortenTestCaseFilterIfRequired(string testCaseFilter)
+        {
+            var maxLength = 256;
+            string shortenTestCaseFilter;
+
+            if (testCaseFilter.Length > maxLength)
+            {
+                shortenTestCaseFilter = testCaseFilter.Substring(0, maxLength) + "...";
```

Comment:
We can use ternary operator here as if else contains single line of code.
`return testCaseFilter.Length > maxLength ? testCaseFilter.Substring(0, maxLength) + "..." : testCaseFilter` OR
`return testCaseFilter.Substring(0, Math.Min(testCaseFilter.Length, maxLength))`

Reply:
For better readability not using terninary operator.

Reply:
That what case ternary operator should be used?
Ternary operators are useful for such scenarios only where if and else conditions have single lines